base/decfp.jl

```diff
@@ -0,0 +1,26 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+# These define the types for IEEE 754-2008 Decimal Floating Point formats
+# Support for these types is available via packages, such as DecFP.jl
+
+export DecimalFloat, DecimalFloatBinary, DecimalFloatPacked,
+       Dec32, Dec64, Dec128, Dpd32, Dpd64, Dpd64
+
+"""Abstract types for all floating point types"""
+abstract DecimalFloat <: AbstractFloat
+abstract DecimalFloatBinary <: DecimalFloat
```

Comment:
I was afraid that that would be to confusing with `BinaryFloat`, and also to put the most important part, i.e. that it is a DecimalFloat, first.